auth-web/src/plugins/vuetify.ts

```diff
@@ -15,6 +15,7 @@ export default new Vuetify({
     },
     themes: {
       light: {
+        primary: '#1669bb',
```

Comment:
Wait - this sets "primary" colour for everyone. Is that OK? If your usage is a special case then I think you should declare a new name (not a built in name).

Reply:
No special case, i believe this is what we want app wide to bring it inline with the rest of our UI's. 
Vuetify primary blue is overridden everywhere else (other ui's) if i'm not mistaken.

Reply:
ok